src/web/app/components/question-types/question-edit-details-form/numericalscale-question-edit-details-form.component.ts

```diff
@@ -0,0 +1,26 @@
+import { Component } from '@angular/core';
+import { FeedbackNumericalScaleQuestionDetails, FeedbackQuestionType } from '../../../../types/api-output';
+import { QuestionEditDetailsFormComponent } from './question-edit-details-form.component';
+
+/**
+ * Question details edit form component for numerical scale question.
+ */
+@Component({
+  selector: 'tm-numericalscale-question-edit-details-form',
```

Comment:
- Let's keep it short, i.e. `numscale` instead of `numericalscale`
- Inconsistency between `numericalscale` and `NumericalScale` (the latter is equivalent to `numerical-scale`. Pick one and follow it consistently.

Reply:
Got it. 👍